crates/swc_ecma_codegen/src/lib.rs

```diff
@@ -3170,6 +3193,9 @@ fn escape_without_source(
             '\u{2029}' => {
                 buf.push_str("\\u2029");
             }
+            '\u{FEFF}' => {
```

Comment:
Why is this case required?

Reply:
Yes, otherwise we will print invalid character:
esbuild - https://github.com/evanw/esbuild/blob/master/internal/js_printer/js_printer.go#L274
terser - https://github.com/terser/terser/blob/master/lib/output.js#L330